projects/vdk-core/tests/functional/run/jobs/pandas-job/10_pandas_step.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+import pandas as pd
+from vdk.api.job_input import IJobInput
+
+
+def run(job_input: IJobInput):
+    # The job only uses job_input.get_managed_connection to check it is initialized properly
+    # Should not use job_input.execute_query
```

Comment:
don't see the reconnect-proof test; where do we check for a reconnect made exactly?

Reply:
added unit test to cover re-connect scenario.